common-model/src/main/java/com/sequenceiq/common/api/cloudstorage/old/GcsCloudStorageV1Parameters.java

```diff
@@ -50,4 +50,10 @@ public int hashCode() {
         return Objects.hash(serviceAccountEmail);
     }
 
+    @Override
+    public String toString() {
+        return "GcsCloudStorageV1Parameters{" +
+                "serviceAccountEmail='" + serviceAccountEmail + '\'' +
```

Comment:
E-mail might be sensitive?

Reply:
this is already in master and maybe 2.36